extensions/amp-access/0.1/amp-login-done-dialog.js

```diff
@@ -208,9 +208,21 @@ export class LoginDoneDialog {
     }
 
     const doc = this.win.document;
-    doc.documentElement.classList.toggle('amp-postback-error', true);
+    doc.documentElement.classList.toggle('amp-error', true);
+    doc.documentElement.setAttribute('data-error', 'postback');
     doc.getElementById('closeButton').onclick = () => {
-      this.win.close();
+      try {
+        this.win.close();
```

Comment:
Just being overprotective here since it's a "catch all" page. Generally, a window will fail closing if it wasn't opened via API (window.open). However, every platform views this very differently: there are very many nuances. So, we are just trying to be the most agnostic to this.